pkg/storage/sources/s3_storage.go

```diff
@@ -138,13 +152,34 @@ func (i *S3Storage) ReadAt(buffer []byte, offset int64) (int, error) {
 	errs := make(chan error, blocks)
 
 	getData := func(buff []byte, off int64) (int, error) {
-		i.lockers[off/int64(i.block_size)].RLock()
-		obj, err := i.client.GetObject(context.TODO(), i.bucket, fmt.Sprintf("%s-%d", i.prefix, off), minio.GetObjectOptions{})
-		i.lockers[off/int64(i.block_size)].RUnlock()
-		if err != nil {
-			return 0, err
+		if i.dummy {
+			atomic.AddUint64(&i.metrics_blocks_r_count, 1)
+			atomic.AddUint64(&i.metrics_blocks_r_bytes, uint64(len(buff)))
+			//atomic.AddUint64(&i.metrics_blocks_r_time_ns, uint64(dtime.Nanoseconds()))
```

Comment:
Should this be commented out still?

Reply:
It's in the dummy branch (so it's instant... no time needs to be added)
00bf562937ee19e728ecb1cf3d9d01050111a7ee